install/jaxrs/bin/ts.jte

```diff
@@ -262,7 +247,7 @@ deploy.delay.in.minutes=5
 ###############################################################
 ## These properties must be set to tell the Test harness the
 ## class names of your porting class implementations.  By default
-## both property sets below point to Sun CI specific classes.
+## both property sets below point to Sun RI specific classes.
```

Comment:
Change RI back to CI

Reply:
For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. issue is from https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jaxrs/bin/ts.jte.jdk11, rectified same.